### ISSUE TITLE
ADD: Change all fees and gas display in $

### ISSUE THREAD

**Issue by realpaaao** [vultisig/vultisig-android#350]
For easier usability change all fees and gas to be displayed in $ Vaule instead. 
Please verify every view and check with figma.

**Comment by johnnyluo** [comment]
Please keep both gas fee in crypto asset , and also in USD value ,  for users have more crypto experience , will expect the gas to display in crypto native way,  for users have less experience , they probably fine with USD value. 

I myself will like gas fee to be displayed in crypto way , so I know whether I paid enough fee for my transaction or now

**Comment by realpaaao** [comment]
@johnnyluo I again agree with you  that we should display both but again was wished by JP see in [fee request](https://discord.com/channels/1203844257220395078/1249417402631061534/1249519593198719006). I will ping you there.

**Comment by realpaaao** [comment]
@yvebe keep as is on swap page and add the same to send screen (see figma):
<img width="898" alt="image" src="https://github.com/vultisig/vultisig-android/assets/167348323/9b4f0ce6-d58e-46f7-9e4b-afe2003b6793">

**Comment by realpaaao** [comment]
@yvebe Yes on verify it makes sense to just have it in USD. I amended figma

**Comment by realpaaao** [comment]
QA done. Opening new issue to implement same on swap screen #1151
